woocommerce/woocommerce
# Add attribute empty state image when no attribute has been added

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Add attribute empty state image when no attribute has been added

Closes #39680 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the new product editor at /wp-admin/admin.php?page=wc-settings&tab=advanced&section=features
2. Go to Product > Add New
3. Go to the Organization tab
4. Check that there's a drawing (empty state) in the Attributes section
5. Make the viewport's width small (mobile)
6. Check that the empty state disappears
7. Return the viewport's width to normal
8. Create an attribute
9. Check that the empty state disappears
10. Delete the attribute
11. Check that it reappears



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
